### ISSUE TITLE
OpenSSL Version Check Error

### ISSUE THREAD

**Issue by Ra2-IFV** [NLnetLabs/unbound#642]
`configure: error: OpenSSL found in /usr/local, but version 0.9.7 or higher is required`  
However  
```
$ /usr/local/bin/openssl version  
OpenSSL 3.0.1 14 Dec 2021 (Library: OpenSSL 3.0.1 14 Dec 2021)
```
How can I fix it?  

System info:Linux ifvlab 5.13.0-30-generic #33-Ubuntu SMP Fri Feb 4 17:03:31 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by sneurlax** [comment]
I am having this same issue, with OpenSSL 1.1.1q I get `configure: error: OpenSSL found in /home/user/dev, but version 0.9.7 or higher is required`.  This error is on both the latest 1.17.0 and 1.16.2.  I've tried using OpenSSL 1.1.1k and 1.1.1s which I've confirmed working elsewhere in our project.

**Comment by sneurlax** [comment]
See also #765, I am following those steps as well

**Comment by mrfuggy** [comment]
> `configure: error: OpenSSL found in /usr/local, but version 0.9.7 or higher is required` However
> 
> ```
> $ /usr/local/bin/openssl version  
> OpenSSL 3.0.1 14 Dec 2021 (Library: OpenSSL 3.0.1 14 Dec 2021)
> ```
I came across the same message. Need exactly OpenSSL 1.x branch, not 3.x branch. Even though the message says "or higher". Need to install OpenSSL 1.x and specify the path to lib more precisely if more than one version is installed.

**Comment by billzhao0315** [comment]
same issue, Do anyone fix it?

**Comment by Ra2-IFV** [comment]
Closed as not occurring on `Ubuntu 24.04.1 LTS`

**Comment by BITljn** [comment]
I came across then same issue on my mac book pro m1, I use ubuntu 24.04 and ubuntu 20.04 on parellels desktop.
